DEPENDENCIES

```diff
@@ -21,6 +21,11 @@ Prerequisites for building the OpenDataPlane (ODP) API
 
    Libraries currently required to link: openssl, libatomic, libconfig
 
+   It is possible to build ODP without OpenSSL by passing flag
+   --without-openssl to configure script. However this will result in
+   ODP-generic not supporting any cryptography algorithm (except NULL) and all
+   random API will return errors. Use at your own risk!
```

Comment:
ODP_RANDOM_BASIC still will be supported. Right?

Reply:
Yes, because we have no way to specify that BASIC is not supported. I'd not dare using it, it depend on the time() during random call, so it is wery basic and wery unsecure.

Reply:
Might it be possible to use /dev/urandom as the randomness provider? I think the Linux implementation if fact is a cryptographically secure PRNG. This will move e.g. export restrictions of cryptograhy to be the Linux distribution's problem instead of a problem of the application linked to ODP.

Reply:
@jmtilli then it might be easier to use `getrandom()`.

Reply:
@muvarov I've updated the file
@jmtilli I would not want to use /dev/urandom at this point. First it might be not available (or might have not enough entropy to generate secure random). Using `getrandom()` syscall might bring additional kernel version dependencies. Let's get this in first and then consider using `getrandom` (maybe even as a separate random implementation) in addition to OpenSSL and null ones.